include/chaiscript/dispatchkit/function_params.hpp

```diff
@@ -36,7 +36,7 @@ namespace chaiscript {
 
```

Comment:
Another issue is with the `std::vector` version of `Function_Params()` constructor. In the case where the vector is empty, it will also cause an assert as it tries to take the address of the `&vec.front()` element -- there is no front! I modified that case locally also to check for empty vector, and return `nullptr` for both `m_begin` and `m_end`.

```c++
      explicit Function_Params(const std::vector<Boxed_Value> &vec)
        : m_begin(vec.empty() ? nullptr : &vec.front()), m_end(vec.empty() ? nullptr : &vec.front() + vec.size())
      {
      }
```